src/js/components/AccordionPanel/doc.js

```diff
@@ -41,6 +46,11 @@ export const themeDoc = {
     type: 'React.Element',
     defaultValue: '<FormDown />',
   },
+  'accordion.pad': {
+    description: 'The pad size around the accordion.',
```

Comment:
The description here isn't accurate, I ran some tests and it looks like the placement of the pad is only wrapping the heading and not the pad of the Accordion itself.
From this description, I would expect the accordion to have a pad, but the pad only applies on the wrapper of the heading and not even on a Panel.

Was that the intention of this PR? I would think the design-system would want the pad to apply on the all Panel.

See an example I created:
![image](https://user-images.githubusercontent.com/6320236/80812437-3687c480-8b85-11ea-9a12-4c12b45ea0c8.png)

Reply:
yes this is around the heading. It is set at 'horizontal: xsmall' which we are not able to change in theme and design system the pad around header is a little big bigger.

Reply:
Cleaned the pad out, resolving the conversation.